content/docs/fedora-releases-guide.md

```diff
@@ -5,17 +5,16 @@ weight: 9
 # How to do Fedora releases with Packit
 
 Let's split the release process into single steps:
-1. New upstream release 
-2. Uploading archive to lookaside cache
-3. Updating dist-git content 
-4. Koji builds
-5. Bodhi updates
-
-Doing Fedora releases with Packit means utilising our 4 jobs:
-1. `propose_downstream`
-2. `koji_build`
-3. `bodhi_update`
-4. `pull_from_upstream`
+1. [New upstream release](#new-upstream-release)
+2. [Uploading archive to lookaside cache](#uploading-archive-to-lookaside-cache)
+3. [Updating dist-git content](#updating-dist-git-content)
```

Comment:
```suggestion
2. [Upload archive to lookaside cache](#upload-archive-to-lookaside-cache)
3. [Update dist-git content](#update-dist-git-content)
```
all other are in passive as well

Reply:
uhhh, can't suggest for unchanged lines -_- up to you, feel free to decline this